students/Obed_Metemey/Session02/PrintGrid.py

```diff
@@ -0,0 +1,55 @@
+
+"""
+/usr/bin/env -- Python3
+Printing a Grid
+Author : OM
+"""
+
+# Part 1
+# This code prints a simple grid
+
+
+# Part 2
+# This code prints a function print_grid(n) that takes one integer argument and prints a grid with the size of the grid given by the argument.
+
+
+def print_grid(n):
+
+    print("+" + "-" * n + "+" + "-" * n + "+")
+    print("|" + " " * n + "|" + " " * n + "|")
+    print("+" + "-" * n + "+" + "-" * n + "+")
+    print("|" + " " * n + "|" + " " * n + "|")
+    print("+" + "-" * n + "+" + "-" * n + "+")
+
+
+# example
+print_grid(4)
+
+
+# Part 3
+# This code draws a similar grid function twith a specified number of rows and columns, and with each cell a given size.
+
+
+def vertical(symbol, spaces):
+    for i in range(symbol):
+        print("| " + " " * spaces, end="")
+    print("|")
+
+
+def horizontal(symbol, spaces):
+    for i in range(symbol):
+        print("+ " + "- " * spaces, end="")
```

Comment:
this is doing spaces two character spaces: "- "

this is bigger, than before, so it doesn't line up...